testsuite/cluster-test/src/main.rs

```diff
@@ -302,8 +316,40 @@ impl BasicSwarmUtil {
             .iter()
             .map(|peer| parse_host_port(peer).unwrap())
             .collect();
+
+        let cluster = Cluster::from_host_port(parsed_peers, &args.mint_file);
+
+        let log_tail_started = Instant::now();
```

Comment:
Can we set up tailer inside `--health-check` action?
We need to make sure we can still run `--swarm --emit-tx` without debug port - we use it when generating load for premainnet, and there is no access to debug port there. With current change we would loose this ability as BasicSwarmUtil here requires debug port to start DebugPortLogThread.